enso/featurize/spacy_features.py

```diff
@@ -53,3 +61,86 @@ def featurize(self, x):
 
     def featurize_batch(self, X, **kwargs):
         return [self.nlp(x).vector for x in X]
+
+@Registry.register_featurizer(ModeKeys.ANY)
+class BayesianesqueFeaturizer(Featurizer):
```

Comment:
Ah these experiments!  I think this is awesome but also think it probably shouldn't live on the master branch, especially given that it looks like the bayesianesque file isn't committed :P